evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -1150,6 +1150,71 @@ def test_pipeline_not_fitted_error(mock_fit, problem_type, X_y_binary, X_y_multi
     clf.feature_importance
 
 
+@pytest.mark.parametrize("problem_type", [ProblemTypes.BINARY, ProblemTypes.MULTICLASS, ProblemTypes.REGRESSION])
+def test_stacked_estimator_in_pipeline(problem_type, X_y_binary, X_y_multi, X_y_regression,
+                                       stackable_classifiers,
+                                       stackable_regressors,
+                                       logistic_regression_binary_pipeline_class,
+                                       logistic_regression_multiclass_pipeline_class,
+                                       linear_regression_pipeline_class):
+    if problem_type == ProblemTypes.BINARY:
+        X, y = X_y_binary
+        base_pipeline_class = BinaryClassificationPipeline
+        input_pipelines = [make_pipeline_from_components([classifier], problem_type) for classifier in stackable_classifiers]
+        parameters = {
+            "Stacked Ensemble Classifier": {
+                "input_pipelines": input_pipelines
+            }
+        }
+        graph = ['Simple Imputer', 'Stacked Ensemble Classifier']
+        comparison_pipeline_class = logistic_regression_binary_pipeline_class
+        objective = 'Log Loss Binary'
+    elif problem_type == ProblemTypes.MULTICLASS:
+        X, y = X_y_multi
+        base_pipeline_class = MulticlassClassificationPipeline
+        input_pipelines = [make_pipeline_from_components([classifier], problem_type) for classifier in stackable_classifiers]
```

Comment:
So this makes a pipeline with a single component, which is the estimator? Where the list of estimators from `stackable_classifiers`/`stackable_regressors` includes all accessible estimators? Cool :)

Reply:
Yup! Maybe overkill but it was my way of checking that any of our accessible classifiers/regressors are allowed as inputs to the stacked ensemble component ahaha